resources/services/glue/workflows.go

```diff
@@ -0,0 +1,256 @@
+				Resolver:    resolveGlueWorkflowTags,
+			},
+			{
+				Name:          "blueprint_details_blueprint_name",
+				Description:   "The name of the blueprint.",
+				Type:          schema.TypeString,
+				Resolver:      schema.PathResolver("BlueprintDetails.BlueprintName"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:          "blueprint_details_run_id",
+				Description:   "The run ID for this blueprint.",
+				Type:          schema.TypeString,
+				Resolver:      schema.PathResolver("BlueprintDetails.RunId"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:        "created_on",
+				Description: "The date and time when the workflow was created.",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:          "default_run_properties",
+				Description:   "A collection of properties to be used as part of each execution of the workflow. The run properties are made available to each job in the workflow",
+				Type:          schema.TypeJSON,
+				IgnoreInTests: true,
+			},
+			{
+				Name:        "description",
+				Description: "A description of the workflow.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "last_modified_on",
+				Description: "The date and time when the workflow was last modified.",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:          "last_run_completed_on",
+				Description:   "The date and time when the workflow run completed.",
+				Type:          schema.TypeTimestamp,
+				Resolver:      schema.PathResolver("LastRun.CompletedOn"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:          "last_run_error_message",
+				Description:   "This error message describes any error that may have occurred in starting the workflow run",
+				Type:          schema.TypeString,
+				Resolver:      schema.PathResolver("LastRun.ErrorMessage"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:          "last_run_name",
+				Description:   "Name of the workflow that was run.",
+				Type:          schema.TypeString,
+				Resolver:      schema.PathResolver("LastRun.Name"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:          "last_run_previous_run_id",
+				Description:   "The ID of the previous workflow run.",
+				Type:          schema.TypeString,
+				Resolver:      schema.PathResolver("LastRun.PreviousRunId"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:          "last_run_started_on",
+				Description:   "The date and time when the workflow run was started.",
+				Type:          schema.TypeTimestamp,
+				Resolver:      schema.PathResolver("LastRun.StartedOn"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:          "last_run_starting_event_batch_condition_batch_size",
+				Description:   "Number of events in the batch.",
+				Type:          schema.TypeBigInt,
+				Resolver:      schema.PathResolver("LastRun.StartingEventBatchCondition.BatchSize"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:          "last_run_starting_event_batch_condition_batch_window",
```

Comment:
Should we avoid repeating batch in this and other columns?

Reply:
Adjusted as suggested, thanks!